Nest/00. Plan.md

```diff
@@ -0,0 +1,67 @@
+# 1 Introduction
+## 1.1 Package manager
+
+## 1.2 Packages and dependencies
+
+## 1.3 Repositories
+
+## 1.4 Package's syntax
+
+# 2 Nest
+## 2.1 Nest's options
+
+## 2.2 Nest commands
+### 2.2.1 Synchronizing the local cache
+### 2.2.2 Troubleshooting
```

Comment:
Troubleshooting what? I know you want me to know from context, I just feel you should be precise.

Reply:
(this applies to other sections with the same name)

Reply:
`Common issues`?

Reply:
"Common issues when using Nest" or "Troubleshooting nest". I didn't mean changing the word itself rather being clear what this is about.

Reply:
`Troubleshooting Nest` wouldn't work because there is troubleshooting sections for almost every command, so the title would need to be more specific, like `Troubleshooting with the synchronization of the local cache with Nest or something like that`, and that is way too long and pretty redundant with the title of the part just before
So I think sticking with simply `Troubleshooting` or `Common issues` is better for those sections

Reply:
Well I think you already know my position: I'd prefer a long and precise title

Reply:
It would be something like "Common issues when synchronizing the local cache using Nest".

Reply:
a long title like this one is pretty redundant with the title of the part just before `Synchronizing the local cache` I don't like having two titles that are so similar

Reply:
Maybe I can simply not detail these `Common issues` parts in the plan, but put them in each command's chapter

Reply:
Yes, that would be more logical actually.